### PR TITLE
feat(select): Use our loading indicator for `isLoading`

### PR DESCRIPTION
**Pull request by evanpurkhiser**
https://user-images.githubusercontent.com/1421724/133173542-e08350b9-05e9-459e-bad1-da39fc5e0c2d.mov

### CODE REVIEW

**Review by davidenwang — APPROVED** [review]
very neat

**Review by dashed — APPROVED** [review]

File: static/app/components/forms/selectControl.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import AsyncCreatable from 'react-select/async-creatable';
 import Creatable from 'react-select/creatable';
 import {withTheme} from '@emotion/react';
 
+import SentryLoadingIndicator from 'app/components/loadingIndicator';
```
Comment (priscilawebdev) [inline static/app/components/forms/selectControl.tsx:15]: shouldn't we keep the name?

File: static/app/components/forms/selectControl.tsx [hunk 2]
```diff
@@ -55,6 +56,10 @@ const MultiValueRemove = (
   </selectComponents.MultiValueRemove>
 );
 
+const LoadingIndicator = () => (
```
Comment (priscilawebdev) [inline static/app/components/forms/selectControl.tsx:59]: can we maybe name it `SelectControlLoadingIndicator`? or move the code to the object?

Comment (EvanPurkhiser) [reply static/app/components/forms/selectControl.tsx]: yep

**Pull request merged**